src/ui/Toast.tsx

```diff
@@ -87,35 +66,43 @@ const VariantToStylesMap = {
 
 type Props = {
   variant: Values<typeof ToastVariants>;
-  header: ReactNode;
   body: string;
   visible?: boolean;
   onClose?: () => void;
 };
 
-const Toast = ({ variant, visible, header, body, onClose }: Props) => {
+const Toast = ({ variant, visible, body, onClose }: Props) => {
+  const icon = useMemo(() => {
+    const icons = {
+      [ToastVariants.PRIMARY]: Icons.PLUS_CIRCLE,
+      [ToastVariants.WARNING]: Icons.EXCLAMATION_CIRCLE,
+      [ToastVariants.DANGER]: Icons.EXCLAMATION_CIRCLE,
+      [ToastVariants.SUCCESS]: Icons.CHECK_CIRCLE,
+    };
+
+    return icons[variant];
+  }, [variant]);
+
   return (
     <BootstrapToast
-      className={`d-inline-block m-1 bg-${variant}`}
+      className={`d-inline-block m-1 p-2`}
```

Comment:
Maybe also add some box shadow on the toast.

Reply:
Not sure I get what you mean, the Bootstrap toasts already come with box shadow, or do you mean some inner box shadow?

Reply:
Aha ok it wasn't clear on the screenshot that there was box-shadow probably because of the grey background ;)